GitHub-Org-Management-Policy.md

```diff
@@ -43,23 +43,27 @@ members to the organization when requested by a Working Group or team.
 
 ## Repositories
 
-Any organization member may request the creation of a new repository within the
-Node.js Foundation GitHub Organization by opening an issue in the Node.js admin
-repository. Provided there are no objections from any TSC or CommComm members,
-such requests are approved automatically after 72 hours. If any objection is
-made, the request may be moved to a vote in each of the Technical Steering and
-Community Committees.  A simple majority of each group *rejecting* the creation
-of the repository is required to block creating the repository. Such requests
-must be posted as issues in the Node.js admin repository.
-
 Any repository created under the Node.js GitHub Organization is considered to be
 a project under the ownership of the Node.js Foundation, and thereby subject
 to the Intellectual Property and Governance policies of the Foundation.
 
-No repository may be deleted, transferred into, or transferred out of the
-Node.js Foundation GitHub Organization without a simple majority of both the
-TSC and CommComm in favor of the action. In certain cases, Node.js Foundation
-Board of Directors approval may also be required.
+Any organization member may request the management of repositories within the
+Node.js Foundation GitHub Organization by opening an issue in the
+[Node.js admin repository][nodejs/admin]. The actions requested could be:
+
+- Creating a new repository
```

Comment:
I had always thought that any member could create a repository... is that not the case?

Reply:
@evanlucas I believe so, just trying to document the process because we generally want people to open an issue before performing the actual action.

Reply:
Also, I am trying to document the whole process in https://github.com/nodejs/admin/pull/68